lib/cheerio.js

```diff
@@ -13,7 +13,8 @@ var api = [
   require('./api/attributes'),
   require('./api/traversing'),
   require('./api/manipulation'),
-  require('./api/css')
+  require('./api/css'),
+  require('./api/serialize')
```

Comment:
Could you rename the new source file to "forms.js"? This matches jQuery's documented category scheme and will make a logical home for the `serializeArray` function in the future.